sdks/python/apache_beam/examples/complete/game/leader_board_test.py

```diff
@@ -65,7 +65,7 @@ def test_leader_board_users(self):
       result = (
           self.create_data(p)
           | leader_board.CalculateUserScores(allowed_lateness=120))
-      assert_that(result, equal_to([]))
+      assert_that(result, equal_to([('user1_team1', 50), ('user2_team2', 2), ('user3_team3', 8), ('user4_team3', 5)]))
```

Comment:
This should not be empty - Prism is handling this correctly, the FnApiRunner was dropping data